core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1001,6 +1001,30 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     assertTrue(assertThrows(classOf[ExecutionException], () => describeResult2.values.get(invalidTopic).get).getCause.isInstanceOf[InvalidTopicException])
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testIncludeDocumentation(quorum: String): Unit = {
+    val consumer = createConsumer()
+    subscribeAndWaitForAssignment(topic, consumer)
+    client = createAdminClient
+
+    val resource = new ConfigResource(ConfigResource.Type.TOPIC, topic)
+    val includeDescribe = new DescribeConfigsOptions().includeDocumentation(true)
+    var describeConfigs = client.describeConfigs(Collections.singletonList(resource), includeDescribe)
+    val pattern = """documentation=([^,]*?)\)""".r
+    var resourceToConfig = describeConfigs.all().get()
+    var matches = pattern.findAllMatchIn(resourceToConfig.toString)
+    var describes = matches.map(e => e.group(1)).toList
+    describes.foreach(e => assertNotNull(e))
```

Comment:
Maybe this should be `assertNotEquals("null", e)`
assertNotNull is used to check the object is null.

Reply:
Yes, you're right, in this case it will the `null` of string literal, not actually `null`